src/main/java/com/spotify/reaper/service/ClusterRepairScheduler.java

```diff
@@ -0,0 +1,143 @@
+package com.spotify.reaper.service;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Throwables;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import com.spotify.reaper.AppContext;
+import com.spotify.reaper.ReaperException;
+import com.spotify.reaper.cassandra.JmxProxy;
+import com.spotify.reaper.core.Cluster;
+import com.spotify.reaper.core.RepairSchedule;
+import com.spotify.reaper.core.RepairUnit;
+import com.spotify.reaper.resources.CommonTools;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+
+public class ClusterRepairScheduler {
+  private static final Logger LOG = LoggerFactory.getLogger(ClusterRepairScheduler.class);
+  private static final String REPAIR_OWNER = "auto-scheduling";
+  private static final String SYSTEM_KEYSPACE_PREFIX = "system";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "SYSTEM_KEYSPACE_PREFIX" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)